snapcraft/common.py

```diff
@@ -30,6 +31,9 @@
 _plugindir = _DEFAULT_PLUGINDIR
 _DEFAULT_SCHEMADIR = '/usr/share/snapcraft/schema'
 _schemadir = _DEFAULT_SCHEMADIR
+_DEFAULT_LIBRARIESDIR = 'usr/share/snapcraft/libraries'
```

Comment:
yeah; I really want to to a massive replace of all the `<something>dir` to be `<something>_dir`